Fundamentals/Setup/Server-Setup/azure-web-apps-v8.md

```diff
@@ -32,11 +32,15 @@ You need to add these `appSettings`:
     value="Examine.LuceneEngine.Directories.SyncTempEnvDirectoryFactory, Examine" />
 ```
 
+You should also add an appsetting `WEBSITE_DISABLE_OVERLAPPED_RECYCLING` with a value of 1 to your Azure WebApp (not this needs to be added to the App configuration via the Azure portal and not via Web.config. This setting prevents Umbraco from starting on an instance before a shutdown is complete - which is important as Umbraco in Azure needs to acquire an exclusive `MainDom` lock in order to start.
+
+You may also want add a setting `WEBSITE_ADD_SITENAME_BINDINGS_IN_APPHOST_CONFIG` with a value of 1 to your Azure WebApp. Although not required, this may help minimise app restarts during deployments. More details can be found in this post: https://ruslany.net/2019/06/azure-app-service-deployment-slots-tips-and-tricks/
```

Comment:
This sounds interesting but I can't find anything about it in the context of Umbraco before. We can't recommend things without really knowing what they do or if they affect Umbraco in any way, I know it only says there's problems with WCF, but that blog also doesn't really know for sure if there may be other problems.

I guess there may actually be usages of WCF with Umbraco 8, so at least that should be warned about. Maybe best to move this to a "worth a try" section in the docs?

Reply:
@nul800sebastiaan having that setting in place is the difference between your production slot being recycled or not when swapped in. Obviously, we really don't want our live slot to be restarted. You are right, in the sense that it is nothing to do with Umbraco - but a really useful Azure tip generally. It does need to be worded better here though, you are right.